package.json

```diff
@@ -20,5 +20,7 @@
     "events": "^1.1.0",
     "imap": "^0.8.16"
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "@types/imap": "^0.8.29"
```

Comment:
This allows VS Code users to get Intellisense based on the TypeScript types. It does not in any way introduce TypeScript to the project or make it a requirement.